libraries/provider_pg_repluser.rb

```diff
@@ -0,0 +1,25 @@
+class Chef
+  class Provider
+    # adds replication user to database
+    class PgRepluser < Chef::Provider::LWRPBase
+      use_inline_resources if defined?(use_inline_resources)
+
+      def whyrun_supported?
+        true
+      end
+
+      action :create do
+        execute 'set-replication-user' do # ~FC009
```

Comment:
Do you need to wrap this in `converge_by` since you're saying `whyrun_supported? will return true`? From [this doc reference](https://docs.chef.io/lwrp.html#whyrun-supported):

> When why-run mode is supported by the a lightweight provider, the converge_by method is used to define the strings that are logged by the chef-client when it is run in why-run mode.

I can't tell if execute already supports 'why run' mode, but maybe it does. If it doesn't though, we'd need to wrap it.

> When a lightweight provider contains only platform resources, the converge_by method is not required because it is already built into all of the platform resources.

Maybe this means it isn't needed after all.